applications/nrf5340_audio/README.rst

```diff
@@ -778,6 +775,17 @@ Selecting the BIS mode
 The CIS mode is the default operating mode for the application.
 You can switch to the BIS mode by adding the ``CONFIG_TRANSPORT_BIS`` Kconfig option set to ``y``  to the :file:`prj.conf` file for the debug version and the :file:`prj_release.conf` file for the release version.
 
+.. _nrf53_audio_app_configuration_select_bidirectional:
+
+Selecting the CIS bidirectional communication
+=============================================
+
+The CIS unidirectional mode is the default operating mode for the application.
+You can switch to the bidirectional mode by adding the :kconfig:option:`CONFIG_STREAM_BIDIRECTIONAL` Kconfig option set to ``y``  to the :file:`prj.conf` file (for the debug version) or to the :file:`prj_release.conf` file (for the release version).
+
+.. note::
+   Only one headset can be connected when using the bidirectional mode
```

Comment:
```suggestion
.. note::
   Only one headset can be connected when using the bidirectional mode.
```
Forgetting a period happens, but it's like missing a ";" in the code ;)

Reply:
Sorry, I want to say I wont do it again, but I make no promises ;) .......